src/shared/components/header/index.js

```diff
@@ -7,17 +7,26 @@ import DesktopNavbar from 'shared/components/navbar/desktop'
 import MobileNavbar from 'shared/components/navbar/mobile'
 import styles from './index.module.css'
 
+const defaultScrollOptions = { offset: -66, align: 'top', duration: 800 }
+
 class Header extends Component {
   constructor () {
     super()
 
     this.state = {
-      isSticky: false
+      isSticky: false,
+      gettingStartedElement: undefined
     }
   }
 
+  componentDidMount () {
+    this.scrollToComponent = require('react-scroll-to-component')
+    this.setState({
+      gettingStartedElement: document.getElementById('gsContainer')
```

Comment:
Since this doesn't affect the render, it shouldn't belong to the state. Simply store it in `this. gettingStartedElement`.

Reply:
Another thing: all the logic behind getting the `gsContainer` by id is "hacky", we could use refs & callbacks instead. But we can track that improvement in another issue.

Reply:
I think that there was a problem with `gatsby` to allow the refs to be passed to the children (page) component. But @PedroMiguelSS can explain it better